invenio_github/api.py

```diff
@@ -323,8 +323,10 @@ def remove_hook(self, repo_id, name):
 
     def repo_last_published_release(self, repo):
         """Retrieves the repository last release."""
+        release_instance = None
         release_object = repo.latest_release(ReleaseStatus.PUBLISHED)
-        release_instance = current_github.release_api_class(release_object)
+        if release_object:
```

Comment:
The release object might not exist since `latest_release` returns an object from `query.first()`. This could lead to problems further down the road. 

I modified the calls to `repo_last_published_release` to either get the release instance or throw an error (usually in the presentation layer throws a 404)